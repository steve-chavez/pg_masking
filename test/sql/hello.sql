create extension if not exists pg_masking;
\echo

select hello();
