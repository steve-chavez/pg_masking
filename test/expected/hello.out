create extension if not exists pg_masking;
\echo

select hello();
 hello 
-------
 hello
(1 row)

